packages/lambdas/src/fhir-to-bundle.ts

```diff
@@ -14,13 +16,19 @@ capture.init();
 // Set by us
 const apiUrl = getEnvOrFail("API_URL");
 const bucketName = getEnvOrFail("BUCKET_NAME");
+const region = getEnvOrFail("AWS_REGION");
+const postHogSecretName = getEnvOrFail("POST_HOG_API_KEY_SECRET");
 
 export async function handler(
   params: ConsolidatedSnapshotRequestSync | ConsolidatedSnapshotRequestAsync
 ): Promise<ConsolidatedSnapshotResponse | void> {
+  const postHogApiKey = await getSecretValue(postHogSecretName, region);
```

Comment:
Not sure if we decided against it, but this seems like one thing that could be initialized outside of the handler so successful invocations that reuse the same lambda environment can benefit from not loading it - since secrets don't change often

Reply:
NVM, let's not early optimize this.